include/Metabox/Posts/CustomTaxonomy.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Metabox to move posts based on category Class.
+ *
+ * @since 2.0.0
+ */
+class BM_Metabox_Posts_CustomTaxonomy extends BM_Metabox_Base {
+
+	protected function initialize() {
+		$this->meta_box_slug = 'bm-posts-by-custom-taxonomy';
+		$this->messages['box_label'] = __( 'Move Posts By Custom Taxonomy', 'bulk-move' );
+		$this->action = 'move_custom_taxonomy';
+	}
+
+	public function render() {
+		?>
+
+		<!-- Custom Taxonomy Start-->
+
+		<fieldset class="options">
+			<table class="optiontable">
+				<tr>
+					<td scope="row" colspan="2">
+						<?php _e( 'Select the post type to show its custom taxonomy.', 'bulk-move' ); ?>
+					</td>
+					<td scope="row">
+				</tr>
+
+				<tr>
+					<td scope="row" colspan="2">
+						<select name="smbm_mbct_post_type" id="smbm_mbct_post_type">
+							<option value="-1"><?php _e( 'Select Post type', 'bulk-move' ); ?></option>
+
+							<?php
+							$custom_post_types = get_post_types( array( 'public' => true ) );
+							?>
+
+							<?php foreach ( $custom_post_types as $post_type ) : ?>
+								<option value="<?php echo esc_attr( $post_type ); ?>"><?php echo esc_html( $post_type ); ?></option>
+							<?php endforeach; ?>
+						</select>
+					</td>
+				</tr>
+
+				<tr class="taxonomy-select-row">
+					<td scope="row" colspan="2">
+						<?php _e( 'Select taxonomy to show its terms.', 'bulk-move' ); ?>
+					</td>
+					<td scope="row">
+				</tr>
+
+				<tr class="taxonomy-select-row">
+					<td scope="row" colspan="2">
+						<select name="smbm_mbct_taxonomy" id="smbm_mbct_taxonomy">
+							<option value="select"><?php _e( 'Select Taxonomy', 'bulk-move' ); ?></option>
+						</select>
+					</td>
+				</tr>
+
+				<tr class="term-select-row">
+					<td scope="row" colspan="2">
+						<?php _e( 'Select terms to move its posts.', 'bulk-move' ); ?>
+					</td>
+					<td scope="row">
+				</tr>
+
+				<tr class="term-select-row">
+					<td scope="row" >
+						<select name="smbm_mbct_selected_term" id="smbm_mbct_selected_term" class="postform">
+							<option class="level-0" value="-1"><?php _e( ' Select Term&nbsp;&nbsp;', 'bulk-move' ); ?></option>
+						</select>
+						==>
+					</td>
+					<td scope="row" >
+						<select name="smbm_mbct_mapped_term" id="smbm_mbct_mapped_term" class="postform">
+							<option class="level-0" value="-1"><?php _e( 'Remove Term&nbsp;&nbsp;', 'bulk-move' ); ?></option>
+						</select>
+					</td>
+				</tr>
+			</table>
+
+			<p class="bm_ct_filters">
+				<?php _e( 'If the post contains other terms, then', 'bulk-move' ); ?>
+				<?php $this->render_overwrite_filters(); ?>
+			</p>
+
+		</fieldset>
+
+		<p class="submit bm_ct_submit">
```

Comment:
Use the `render_submit` helper for this.

Reply:
> Use the `render_submit` helper for this.

Fixed